lib/blocking-io.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function () {
+  if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
+    process.stdout._handle.setBlocking(true)
+    process.stderr._handle.setBlocking(true)
+  }
```

Comment:
Looks like it was added in v0.11.2: https://github.com/nodejs/node/commit/20176a98416353d4596900793f739d5ebf4f0ee1 in response to https://github.com/nodejs/node-v0.x-archive/issue/3584